### DISCUSSION TITLE
Add ReplayGain support (volume normalization)

### DISCUSSION THREAD

**Discussion by jgandert** [PaulWoitaschek/Voice#1372] [Ideas]
There was a [feature request in 2015](https://github.com/PaulWoitaschek/Voice/issues/223) where apparently there were problems with the implementation. Maybe things have improved or there can be a work around. At least it should be possible to read ReplayGain tags and automatically apply a gain change.

**Comment by PaulWoitaschek** [comment]
The linked thread is about gain. That is implemented.

For replaygain we would need to parse the file headers and constantly adjust the volume:
https://de.wikipedia.org/wiki/Replay_Gain

Up for grabs :P
